git/intermediate_git/a_deeper_look_at_git.md

```diff
@@ -221,10 +216,7 @@ This section contains helpful links to related content. It isn’t required, so
 ### Knowledge check
 
 This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
-
+* <a class='knowledge-check-link' href='https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell'>Explain what it means for branches to be pointers.</a>
```

Comment:
You'll need to add a newline before the first bullet point here otherwise it [won't format properly](https://www.theodinproject.com/lessons/preview?uuid=ba385bbb-95eb-413a-a6e1-d3aaaa9eb7c8).